src/client/src/components/DependencyInfo/index.tsx

```diff
@@ -0,0 +1,81 @@
+import * as React from "react";
+import styles from "./styles.module.css";
+import * as getSvg from "../../utils/getSvgUrl";
+import classnames from "classnames";
+import { InjectedIntlProps, injectIntl, defineMessages } from "react-intl";
+
+interface IDependencyInfoProps {
+  frameworkName: string;
+  installed: boolean;
+  intl: any;
+}
+
+type Props = IDependencyInfoProps;
+
+const messages = defineMessages({
+  installed: {
+    id: "dependencyChecker.installed",
+    defaultMessage: " detected!"
+  },
+  notInstalled: {
+    id: "dependencyChecker.notInstalled",
+    defaultMessage: " not detected. Click to install."
+  }
+});
+
+/*
+ * Props:
+ * - frameworkName: "NodeJS" | "Flask"
+ * - installed: boolean, true if correct version installed, otherwise false
+ */
+class DependencyInfo extends React.Component<Props> {
```

Comment:
This is a new component, so a great candidate for unit tests as well :)

Reply:
okay, unit test will be added after I do the refactoring